internal/exec/resolvable/resolvable.go

```diff
@@ -61,12 +63,12 @@ func (*Object) isResolvable() {}
 func (*List) isResolvable()   {}
 func (*Scalar) isResolvable() {}
 
-func ApplyResolver(s *types.Schema, resolver interface{}) (*Schema, error) {
+func ApplyResolver(s *types.Schema, resolver interface{}, directives map[string]directives.ResolverVisitor) (*Schema, error) {
```

Comment:
Again what type of directives? These are only on `FIELD_DEFINITION`. What if we add more types?

Reply:
Technically, this is an internal API, so we can change it/break it.